internal/functions/serve/serve.go

```diff
@@ -22,6 +22,24 @@ const (
 	customDockerImportMapPath = "/home/deno/import_map.json"
 )
 
+func ParseEnvFile(envFilePath string) ([]string, error) {
+	env := []string{}
+	if len(envFilePath) == 0 {
+		return env, nil
+	}
+	envMap, err := godotenv.Read(envFilePath)
+	if err != nil {
+		return env, err
+	}
+	for name, value := range envMap {
+		if strings.HasPrefix(name, "SUPABASE_") {
+			return env, errors.New("Invalid env name: " + name + ". Env names cannot start with SUPABASE_.")
+		}
+		env = append(env, name+"="+value)
+	}
+	return env, nil
+}
+
 func Run(ctx context.Context, slug string, envFilePath string, verifyJWT bool, importMapPath string, fsys afero.Fs) error {
```

Comment:
Looks like we missed to update `cmd/functions.go` with the new flag?

Reply:
It's using the same `--env-file` flag that we added previously :) Previously it was only exposed to one exec command, but now it's for the whole container so deno env vars can be defined inside too.